src/js/components/Table.js

```diff
@@ -120,35 +133,64 @@ export default class Table extends Component {
     }
   }
 
+  _adjustBodyCells () {
+    //adjust table body cells to have link to the header
+    //so that in responsive mode it displays the text as content in css
+    let headerCells = this.refs.table.querySelectorAll('thead th');
+    if (headerCells.length > 0) {
+      let rows = this.refs.table.querySelectorAll('tbody tr');
+
+      [].forEach.call(rows, (row) => {
+        [].forEach.call(row.cells, (cell, index) => {
+          cell.setAttribute('data-th', headerCells[index].innerText);
+        });
+      });
+    }
+  }
+
   _onResize () {
     this._alignMirror();
+    this._layout();
   }
 
-  _buildMirror () {
-    var tableElement = this.refs.table;
-    var cells = tableElement.querySelectorAll('thead tr th');
-    var mirrorElement = this.refs.mirror;
-    var mirrorRow = mirrorElement.querySelectorAll('thead tr')[0];
-    while (mirrorRow.hasChildNodes()) {
-      mirrorRow.removeChild(mirrorRow.lastChild);
+  _layout () {
+    let availableSize = this.refs.container.offsetWidth;
+    let numberOfCells = this.refs.table.querySelectorAll('thead th').length;
+
+    if ((numberOfCells * MIN_CELL_WIDTH) > availableSize) {
```

Comment:
Good idea on the mirror evaluation. But that would only work for scrollable tables right?

Initially I added support for the table be responsive, regardless of the scrollable option.